### PR TITLE
[OBSDEF-24807] make breadcrumbs design as per DDS

### PR DESCRIPTION
**Pull request by Bhatii**
## Description
Implementation of Collapsible breadcrumbs functionality

##### isCollapsible:
- Check if the breadcrumbs should be collapsible or not
- Default: False
- If set to True, breadcrumbs trail will be truncated based on the maxItems prop
image
##### maxItems:
- Max breadcrumb items count, if breadcrumb count passes this threshold, breadcrumb body truncates into collapsed form 
- Default: 3
- _isCollapsible_ should be true for breadcrumbs collapse functionality to activate
image
image

## Before
image

## After
image

### Test Environment
http://10.249.253.4:6006/?path=/story/breadcrumb--default-breadcrumbs

### CODE REVIEW

File: src/styles/components/Breadcrumb.scss [hunk 1]
```diff
@@ -12,41 +12,88 @@
...
+
+    @include breadcrumb-item($blue-600);
+
+    :hover {
+      color: $blue-700;
+    }
+  }
+
+  &.active-element {
+    font-size: 24px;
```
Comment (maheshkulkarni4) [inline src/styles/components/Breadcrumb.scss:95]: Do you think we can keep this in rems?  Also, could you check if we have any variables already covering Font-size? 
The font Size, Padding, margins 
This applies to all font sizes used in this stylesheet, check if we can fix all

File: src/styles/components/Breadcrumb.scss [hunk 2]
```diff
@@ -12,41 +12,88 @@
...
+.breadcrumbs-container {
+  margin: 24px 0;
+}
  .breadcrumb {
    display: flex;
    flex-wrap: wrap;
    width: fit-content;
-   padding: 0 0;
+   padding: 0 2px;
    margin-bottom: 1rem;
```
Comment (maheshkulkarni4) [inline src/styles/components/Breadcrumb.scss:23]: Lets keep things consistent.  Fonts should be in REMs only
padding, margin - REMs preferred, but ok if we want px too.  But keep it consistent in the page.

**Review by maheshkulkarni4 — APPROVED** [review]
LGTM, Take care of the comments

File: src/styles/components/Breadcrumb.scss (hunk 1 above)
Comment (Bhatii) [reply src/styles/components/Breadcrumb.scss]: Done

File: src/styles/components/Breadcrumb.scss (hunk 2 above)
Comment (Bhatii) [reply src/styles/components/Breadcrumb.scss]: Updated to rem

**Review by VarunAttarde22 — APPROVED** [review]

**Review by amaurya1318050 — APPROVED** [review]

**Pull request merged**